charts/backbeat/templates/lifecycle/conductor_deployment.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "backbeat.fullname" . }}-lifecycle-conductor
+  labels:
+    app: {{ template "backbeat.name" . }}-lifecycle
+    chart: {{ template "backbeat.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.lifecycle.conductor.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "backbeat.name" . }}-lifecycle
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ template "backbeat.fullname" . }}-lifecycle-conductor
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/usr/src/app/docker-entrypoint.sh"]
+          args: ["npm", "run", "lifecycle_conductor"]
+          env:
+            - name: ZOOKEEPER_AUTO_CREATE_NAMESPACE
+              value: "1"
+            - name: ZOOKEEPER_CONNECTION_STRING
+              value: "{{- printf "%s-zenko-quorum:2181" .Release.Name | trunc 63 | trimSuffix "-" -}}"
+            - name: KAFKA_HOSTS
+              value: "{{- printf "%s-kafka-0.%s-kafka:9092" .Release.Name .Release.Name | trunc 63 | trimSuffix "-" -}}"
+            - name: MONGODB_HOSTS
+              value: "{{ .Release.Name }}-mongodb-replicaset-0.{{ .Release.Name }}-mongodb-replicaset:27017,{{ .Release.Name }}-mongodb-replicaset-1.{{ .Release.Name }}-mongodb-replicaset:27017,{{ .Release.Name }}-mongodb-replicaset-2.{{ .Release.Name }}-mongodb-replicaset:27017"
+            - name: AUTH_TYPE
+              value: service
+            - name: AUTH_ACCOUNT
+              value: service-lifecycle
+            - name: EXTENSIONS_LIFECYCLE_ZOOKEEPER_PATH
+              value: "{{ .Values.lifecycle.zookeeper.path }}"
+            - name: EXTENSIONS_LIFECYCLE_BACKLOG_METRICS_ZKPATH
```

Comment:
What's the relation between metrics and Zookeeper?

Reply:
The name is taken from the backbeat config file 
```
"extensions": {
    "lifecycle": {
        ...
        "backlogMetrics": {
            "zkPath": "/lifecycle/run/backlog-metrics",
            ....
        },
    },
}
```